db/schema.rb

```diff
@@ -292,7 +292,9 @@
     t.datetime "updated_at", precision: 6, null: false
     t.boolean "current_address_matches_id", comment: "True if the user indicates that their current address matches the address on the ID they're bringing to the Post Office."
     t.jsonb "selected_location_details", comment: "The location details of the Post Office the user selected (including title, address, hours of operation)"
+    t.string "unique_id", comment: "Unique ID to use with the USPS service"
```

Comment:
Perhaps in the same ticket as my other comments mentions: can w come back later and make this a required field once we've generated unique IDs for all records?

Reply:
Follow up ticket created: https://cm-jira.usa.gov/browse/LG-7024